.github/workflows/push.yml

```diff
@@ -49,30 +50,33 @@ jobs:
           source venv/bin/activate
           python -m pip install --upgrade pip
           pip3 install -r requirements.txt
-          pip install pytest-cov
+          pip install pytest-cov coverage
           tests/bin/error_code_report.py
           tests/bin/test_db_config_setup.py
           sudo systemctl start mysql.service
           sudo mysql -u${{ env.MYSQL_ROOT_USERNAME }} -p${{ env.MYSQL_ROOT_PASSWORD }} -e "CREATE DATABASE ${{ env.DATABASE_NAME }};"
           sudo mysql -u${{ env.MYSQL_ROOT_USERNAME }} -p${{ env.MYSQL_ROOT_PASSWORD }} -e "CREATE USER '${{ env.DATABASE_USERNAME }}'@'localhost' IDENTIFIED BY '${{ env.DATABASE_PASSWORD }}';"
           sudo mysql -u${{ env.MYSQL_ROOT_USERNAME }} -p${{ env.MYSQL_ROOT_PASSWORD }} -e "GRANT ALL PRIVILEGES ON ${{ env.DATABASE_NAME }}.* TO '${{ env.DATABASE_USERNAME }}'@'localhost';"
           sudo mysql -u${{ env.MYSQL_ROOT_USERNAME }} -p${{ env.MYSQL_ROOT_PASSWORD }} -e "FLUSH PRIVILEGES;"
-          pytest --cov=. --cov-report=xml --cov-report=term-missing --junitxml=junit.xml tests/switchmap_
+          pytest --cov=switchmap --cov-report=lcov --cov-report=term-missing tests/switchmap_
+          
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v5
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           slug: PalisadoesFoundation/switchmap-ng
           verbose: true
+          files: ./coverage.lcov
```

Comment:
1. Please put this in a `coverage/` directory
3. Add `coverage/` to `.gitignore`
5. We need less clutter in the root directory.
6. This is what we did in Talawa-Admin

When complete, this can be merged